app/controllers/documents_controller.rb

```diff
@@ -24,6 +24,7 @@ def show
   def update
     document = Document.find_by_param(params[:id])
     document.update!(update_params(document))
+    TimelineEntry.create!(document: document, user: current_user, entry_type: "updated_content")
```

Comment:
I noticed that the 'create document' workflow means we end up with a 'created' and an 'updated' entry in the history. Would that confuse people? It did surprise me a bit.

Reply:
There's also a new entry after 'change content' -> 'save' when nothing has changed.

Reply:
> I noticed that the 'create document' workflow means we end up with a 'created' and an 'updated' entry in the history. Would that confuse people? It did surprise me a bit.

I don't know, but I'd like to keep separate it so that we can attach different data to it.

> There's also a new entry after 'change content' -> 'save' when nothing has changed.

Why would you press save if nothing changed?

Reply:
I rarely press cancel in general. Reasoning: I don't care if I haven't changed anything.